# Configuration file for the microphone
def:
  user_id : "micol_testing"
  bot_name: "QTRobotBot"
  bot_alias: "qt_robot_demo" 
  region_name: "us-west-2"
